### PR TITLE
relay token register and lock events

### PR DESCRIPTION
**Pull request by xiaoch05**

### CODE REVIEW

File: .maintain/config/ropsten_crab.sample.yml [hunk 1]
```diff
@@ -30,6 +30,11 @@ ethereum:
       topics:
         # set authorities topic
         - 0x91d6d149c7e5354d1c671fe15a5a3332c47a38e15e8ac0339b24af3c1090690f
+    backing:
+      address: "0x46433e69e0c3c3D7BD0601490a6150A592412E80"
+      topics:
+        - "0x5c73cd563b262a0c464bed44643bde46bdc91fec8bc5ce5dce7152ac067d80c6"
+        - "0x9e91c651fb786188899553f59c3b822856be7f6d8b9d20719fa7ec404310dade"
```
Comment (HackFisher) [inline .maintain/config/ropsten_crab.sample.yml:37]: Put it in https://github.com/darwinia-network/bridger/blob/master/.maintain/config/ropsten_pangolin.sample.yml instead,

ropsten_crab will no longer be used.

Comment (xiaoch05) [reply .maintain/config/ropsten_crab.sample.yml]: fixed by https://github.com/darwinia-network/bridger/pull/167/commits/9ecca161b153d7bf04f8a5c20a0e843cf5933de7

**Comment by HackFisher** [comment]
@xiaoch05 Would you please create/link the issue for the subxt's version problem which prevent from this to be used in test/production.

And please give the workaround solution you have if there is.

**Comment by xiaoch05** [comment]
> @xiaoch05 Would you please create/link the issue for the subxt's version problem which prevent from this to be used in test/production.
> 
> And please give the workaround solution you have if there is.

the pr https://github.com/paritytech/substrate/pull/7640 is not merged into the latest version v3.0.0, but darwinia-common used the master branch include this change.
subxt uses version 3.0.0, so it can't parse the new response of API `getBlock`.

this pr has no effect on this problem.
also, the latest version of darwinia-common can't work with bridger's master branch.

**Comment by xiaoch05** [comment]
> > @xiaoch05 Would you please create/link the issue for the subxt's version problem which prevent from this to be used in test/production.
> > And please give the workaround solution you have if there is.
> 
> the pr [paritytech/substrate#7640](https://github.com/paritytech/substrate/pull/7640) is not merged into the latest version v3.0.0, but darwinia-common used the master branch include this change.
> subxt uses version 3.0.0, so it can't parse the new response of API `getBlock`.
> 
> this pr has no effect on this problem.
> also, the latest version of darwinia-common can't work with bridger's master branch.

@HackFisher We have given a  temporary solution by commit https://github.com/wuminzhe/substrate-subxt/commit/c9163befd3bdf3062bdb4912c50933083ff69eba

**Comment by HackFisher** [comment]
@xiaoch05 substrate-subxt's git commit in Cargo.lock should be update to https://github.com/wuminzhe/substrate-subxt/commit/c9163befd3bdf3062bdb4912c50933083ff69eba

**Comment by xiaoch05** [comment]
> > > @xiaoch05 Would you please create/link the issue for the subxt's version problem which prevent from this to be used in test/production.
> > > And please give the workaround solution you have if there is.
> > 
> > 
> > the pr [paritytech/substrate#7640](https://github.com/paritytech/substrate/pull/7640) is not merged into the latest version v3.0.0, but darwinia-common used the master branch include this change.
> > subxt uses version 3.0.0, so it can't parse the new response of API `getBlock`.
> > this pr has no effect on this problem.
> > also, the latest version of darwinia-common can't work with bridger's master branch.
> 
> @HackFisher We have given a temporary solution by commit [wuminzhe/substrate-subxt@c9163be](https://github.com/wuminzhe/substrate-subxt/commit/c9163befd3bdf3062bdb4912c50933083ff69eba)



> @xiaoch05 substrate-subxt's git commit in Cargo.lock should be updated to [wuminzhe/substrate-subxt@c9163be](https://github.com/wuminzhe/substrate-subxt/commit/c9163befd3bdf3062bdb4912c50933083ff69eba)

fixed by https://github.com/darwinia-network/bridger/pull/167/commits/486fd64b1460ce09460b1def4a4fcbb53f47bbe1

**Review by HackFisher — APPROVED** [review]

File: primitives/src/byte.rs [hunk 2]
```diff
@@ -16,9 +16,9 @@ macro_rules! hex {
 		let mut s = String::new();
 		for i in $bytes {
 			s.push_str(&format!("{:02x}", i));
-		}
-		s
-	}};
+			}
```
Comment (AurevoirXavier) [inline primitives/src/byte.rs:19]: These format looks weird

**Pull request merged**